include/internal/dso.h

```diff
@@ -163,71 +164,8 @@ DSO *DSO_dsobyaddr(void *addr, int flags);
  */
 void *DSO_global_lookup(const char *name);
 
-/* BEGIN ERROR CODES */
-/*
- * The following lines are auto generated by the script mkerr.pl. Any changes
- * made after this point may be overwritten when the script is next run.
- */
-
 int ERR_load_DSO_strings(void);
```

Comment:
because it *was* in foo.h I kept it there.  All the conditional guards are in foo.h and not replicated into fooerr.h (look at engine.h and engineerr.h for example).